### PR TITLE
Cats 1560

### PR DESCRIPTION
**Pull request by getbez**

### CODE REVIEW

File: app/assets/javascripts/regional_request.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```
Comment (asuraphel) [inline app/assets/javascripts/regional_request.js]: This seems to be added by mistake...

File: app/controllers/receipts_controller.rb [hunk 2]
```diff
@@ -1,12 +1,38 @@
 class ReceiptsController < ApplicationController
 
     def index 
-        @receipts = Receipt.all 
+
```
Comment (asuraphel) [inline app/controllers/receipts_controller.rb]: Beza, I did this feature as well but it's showing as new. What went wrong? :)

**Review by asuraphel — CHANGES_REQUESTED** [review]
@getbez I saw some discrepancies with some of the files. Please handle them.

File: app/assets/javascripts/regional_request.js (hunk 1 above)
Comment (getbez) [reply app/assets/javascripts/regional_request.js]: I had to recover my repo after a sudden corruption. This seems to have come from your commit 
[https://github.com/fmip/cats-web/commit/1c24bfcac735b967fb615362f08f021c5dafee9a](url) . We can remove the comments. Same reason for your other comment.

**Review by getbez — COMMENTED** [review]
@asuraphel  I have reviewed this commit again and no current code has been affected. Most changes are due to added modules. The others are files i have been working with. The two files you mentioned were shown because of merging after reseting my HEAD to the last develop verison.

**Review by asuraphel — APPROVED** [review]

**Pull request merged**